utils/actions/shopActions.ts

```diff
@@ -17,3 +19,21 @@ export const getStoreProducts = async (slug: string): Promise<Product[]> => {
 
   return storeProducts
 }
+
+export const getStoreStory = async (
```

Comment:
adding new methods here to the `shopActions` file since the user authentication works differently, as a customer browsing the site I should be able to see any store's story or products.